### ISSUE TITLE
Audio's text alternatives is missing

### ISSUE THREAD

**Issue by ecureuill** [solex2006/SELIProject#30]
## Expected Behavior

![image](https://user-images.githubusercontent.com/993369/76174010-353cab80-6183-11ea-8d11-aeb6e2068b9b.png)

## Current Behavior

![image](https://user-images.githubusercontent.com/993369/76175574-15f64c00-618c-11ea-9940-b13c96df44a0.png)

## Possible Implementation

For audio-only (no video) sign language is not required. 
Please refer to Tasks in requirement #8

**Comment by ecureuill** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [BUG_XXX-## Title](https://trello.com/c/w1KGQQcW/15-bugxxx-title)

**Comment by CAndresH** [comment]
Camila the change has been made:
![image](https://user-images.githubusercontent.com/50594090/77075483-66f21580-69c0-11ea-962e-2e086f3f1f45.png)
